changelog/unreleased/2623.md

```diff
@@ -0,0 +1,4 @@
+VAST has a new configuration setting, `vast.zstd-compression-level`, to control
+the compression level of the Zstd algorithm used in both the Feather and
+Parquet store backend. The default value is taken from Apache Arrow, which
+changes the default for Parquet, which previously had its own default of `9`.
```

Comment:
Suggestion for the last sentence to avoid the double "which":

> The default level is set by the Apache Arrow library, and for Parquet is no longer explicitly defaulted to 9.